notebooks/supernovae_plus_extended_source_tutorial.ipynb

```diff
@@ -0,0 +1,245 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.        "/Users/narayankhadka/slsim/tests/TestData/psf_kernels_for_deflector.npy"
</pre><p>make this a relative path to SLSim such that it does not need to be changed by other users</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LSST-strong-lensing/slsim/pull/130/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>